supported_interpreters.md

```diff
@@ -0,0 +1,280 @@
+      <br/>
+      <a href="https://hive.apache.org/" target="_blank">Hive</a>,
+      <a href="https://phoenix.apache.org/" target="_blank">Phoenix</a>,
+      <a href="https://drill.apache.org/" target="_blank">Drill</a>,
+      <a href="http://tajo.apache.org/" target="_blank">Tajo</a> 
+      are available
+    </td>
+    <td>
+      <a href="http://www.postgresql.org/" target="_blank">PostgreSQL</a>,
+      <a href="https://www.mysql.com/" target="_blank">MySQL</a>,
+      <a href="https://mariadb.org/" target="_blank">MariaDB</a>,
+      <a href="https://aws.amazon.com/documentation/redshift/" target="_blank">Redshift</a>,
+      <br/>
+      <a href="https://hive.apache.org/" target="_blank">Hive</a>,
+      <a href="https://phoenix.apache.org/" target="_blank">Phoenix</a>,
+      <a href="https://drill.apache.org/" target="_blank">Drill</a>,
+      <a href="http://tajo.apache.org/" target="_blank">Tajo</a> 
+      are available
+    </td>
+    <td>N/A</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>
+      <h6><a href="https://cloud.google.com/bigquery/" target="_blank">BigQuery</a></h6>
+    </td>
+    <td>O</td>
+    <td>N/A</td>
+    <td>N/A</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>
+      <h6><a href="https://www.python.org/" target="_blank">Python</a></h6>
+    </td>
+    <td>O</td>
+    <td>O</td>
+    <td>N/A</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>
+      <h6><a href="http://livy.io/" target="_blank">Livy</a></h6>
+    </td>
+    <td>O</td>
+    <td>O</td>
+    <td>N/A</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>
+      <h6><a href="https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html" target="_blank">HDFS</a></h6>
+    </td>
+    <td>O</td>
+    <td>O</td>
+    <td>N/A</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>
+      <h6><a href="http://www.alluxio.org/" target="_blank">Alluxio</a></h6>
+    </td>
+    <td>O</td>
+    <td>O</td>
+    <td>N/A</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>
+      <h6><a href="https://hbase.apache.org/" target="_blank">Hbase</a></h6>
+    </td>
+    <td>O</td>
+    <td>O</td>
+    <td>N/A</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>
+      <h6><a href="https://github.com/twitter/scalding" target="_blank">Scalding</a></h6>
+      <span style="font-size:75%">Local mode only, not included in binary package</span>
+    </td>
+    <td>O</td>
+    <td>O</td>
+    <td>O</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>
+      <h6><a href="https://www.elastic.co/products/elasticsearch" target="_blank">Elasticsearch</a></h6>
+    </td>
+    <td>O</td>
+    <td>O</td>
+    <td>O</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td>
+      <h6><a href="https://angularjs.org/" target="_blank">Angular</a></h6>
+    </td>
+    <td>O</td>
+    <td>O</td>
+    <td>O</td>
+    <td>O</td>
+  </tr>
+  <tr>
+    <td>
+      <h6><a href="https://daringfireball.net/projects/markdown/" target="_blank">Markdown</a></h6>
```

Comment:
Maybe better to use https://code.google.com/archive/p/markdown4j/ instead since Zeppelin uses markdown4j library by default.